### PR TITLE
vm-image: add sqlexporter for autoscaling metrics

### PR DESCRIPTION
**Pull request by skyzh**
## Problem

As discussed in https://github.com/neondatabase/autoscaling/pull/895, we want to have a separate sql_exporter for simple metrics to avoid overload the database because the autoscaling agent needs to scrape at a higher interval. The new exporter is exposed at port 9499. I didn't do any testing for this pull request but given it's just a configuration change I assume this works.

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics? if so did you add the relevant metrics to the dashboard?
- [ ] If this PR requires public announcement, mark it with /release-notes label and add several sentences in this section.

## Checklist before merging

- [ ] Do not forget to reformat commit message to not include the above checklist

### CODE REVIEW

File: vm-image-spec.yaml [hunk 1]
```diff
@@ -16,7 +16,11 @@ commands:
   - name: sql-exporter
     user: nobody
     sysvInitAction: respawn
-    shell: '/bin/sql_exporter -config.file=/etc/sql_exporter.yml'
+    shell: '/bin/sql_exporter -config.file=/etc/sql_exporter.yml -web.listen-address=:9399'
```
Comment (sharnoff) [inline vm-image-spec.yaml:19]: changing this line looks sketchy to me (is it the same port before and after?) but everything else lgtm

(if it's worth being explicit about the listen address for the "normal" sql-exporter, may be better to do that in a separate change. idk - ultimately that part is probably owned by compute, so if you feel this is right, go for it)

File: vm-image-spec.yaml [hunk 2]
```diff
@@ -16,7 +16,11 @@ commands:
...
+  - name: sql-exporter-autoscaling
```
Comment (sharnoff) [inline vm-image-spec.yaml:20]: Worth considering if this should be called "autoscaling" or just "lfc" — i.e. "who it's for" vs "what it is"

Could go either way, I have no preference, just putting it out there.

**Review by sharnoff — APPROVED** [review]
re: testing- it would be good to test on staging, using a compute image built from CI. Feel free to ping me for details if unsure how.

File: vm-image-spec.yaml (hunk 1 above)
Comment (skyzh) [reply vm-image-spec.yaml:19]: It's the same port and i'd like to make it explicit. I can separate another PR.

Comment (sharnoff) [reply vm-image-spec.yaml:19]: Your call — either way, I'd test on staging that the metrics still show up in the console (or are available via the "per-VM metrics" dashboard in grafana)

Comment (skyzh) [reply vm-image-spec.yaml:19]: Yep, I’ll merge after next week’s platform release to have enough time testing it in staging.

**Comment by skyzh** [comment]
tested in staging:

```
> curl localhost:9499/metrics
# HELP lfc_approximate_working_set_size Approximate working set size in pages of 8192 bytes
# TYPE lfc_approximate_working_set_size gauge
lfc_approximate_working_set_size 269
# HELP lfc_cache_size_limit LFC cache size limit in bytes
# TYPE lfc_cache_size_limit gauge
lfc_cache_size_limit 6.30194176e+08
# HELP lfc_hits lfc_hits
# TYPE lfc_hits gauge
lfc_hits 0
# HELP lfc_misses lfc_misses
# TYPE lfc_misses gauge
lfc_misses 279
# HELP lfc_used LFC chunks used (chunk = 1MB)
# TYPE lfc_used gauge
lfc_used 66
# HELP lfc_writes lfc_writes
# TYPE lfc_writes gauge
lfc_writes 279
```

**Pull request merged**